mkosi/__init__.py

```diff
@@ -318,13 +318,12 @@ def __str__(self) -> str:
         return self.name
 
     @classmethod
-    def from_string(cls, name: str) -> "OutputFormat":
+    def Format(cls, name: str) -> "OutputFormat":
         """A convenience method to be used with argparse"""
         try:
             return cls[name]
         except KeyError:
-            # this let's argparse generate a proper error message
-            return name  # type: ignore
+            raise ValueError
```

Comment:
Shouldn't this be `raise ValueError()` so we raise an instance of `ValueError` instead of the type itself?

Reply:
That's the same. There's special support for `raise <type>`.